CHANGELOG.md

```diff
@@ -79,6 +79,11 @@ The function for calling sns can now call any bundled binary.
 - Ledger for financial integration testing
 - Internet Identity for user registration and authenttication
 
+### feat(frontend-canister): Add simple redirects from `<asset>` to `<asset>.html` and `<asset>/index.html`
+
+The asset canister now by default redirects any request to `<asset>` to `<asset>.html`.
+This can be disabled by setting the field `"redirect"` to `false` in the canister settings in dfx.json.
```

Comment:
This still references `redirect`, but should talk about the `enable_aliasing` property in ic-assets.json5